GoogleForCreators/web-stories-wp
# Fix editor crashing while using composed characters on Text Element

## Summary

It only fixes editor crashing when using text elements with composed characters. Additional bugs related to the usage of composed characters will be fixed in follow-up PRs (e.g. #2002)

## Relevant Technical Choices

- This PR only avoids consuming `getStyle` from a node that not exists.
- The test added only ensure that we can render normally composed characters

## User-facing changes

It will avoid breaking the editor. It also allows use composed characters in some ways (pressing <kbd>enter</kbd> or a non-composed character before leaving the edit mode on the text element), but there are still issues to improve the usage of composed characters.

## Testing Instructions

- Change your computer to Japanese (Hiragana) [MacOS process]
- On Web Stories Editor, add a new Text Element
- Enter on Edit Mode of the Text Element added previously and type `konnichi` in your QWERTY keyboard, which is expected to render こんにち on the text element editable area.
- Press <kbd>enter</kbd>
- Click outside to leave the Edit Mode of the Text Element

<img src="https://user-images.githubusercontent.com/1680157/82942003-b11de700-9f6d-11ea-98ad-c5cb80dc1c66.gif" width="60%">

---



Solving #1836
